src/lib/config.js

```diff
@@ -0,0 +1,46 @@
+import FaTwitter from 'react-icons/lib/fa/twitter';
+import FaGithub from 'react-icons/lib/fa/github';
+import { h } from 'preact';
+
+const imageProps = {
+  fill: '#444',
+  width: 24,
+  height: 24,
+};
+
+const config = {
+  meetupApi:
+    'https://api.meetup.com/require-lx/events?key=6161504e80005a355e671d2fb5d2e&page=20&sign=true',
```

Comment:
I would make this `meetup` part of the config file a bit more reusable, but it's fine for now.